.github/workflows/quic.yml

```diff
@@ -259,7 +261,9 @@ jobs:
     if: ${{ (github.event_name == 'repository_dispatch' && github.event.client_payload.pr == '') || inputs.commit }}
     strategy:
       fail-fast: false
-    runs-on: 'ubuntu-20.04'
```

Comment:
All of these don't need to run on multiple. Keep them on 20.04. They're just processing data